frontend/assets/stylesheets/components/_callout.scss

```diff
@@ -0,0 +1,39 @@
+.call-out-container {
```

Comment:
This should be the root component if `.call-out` is dependant on that container (`.call-out` becomes `.call-out__content`)